src/chat/card-service.js

```diff
@@ -105,20 +93,45 @@ class CardService extends EventEmitter {
         this.owner.cards.push(new Card(strDate, [], false));
     }
 
-    /**
-     * @description
-     * Restore shopping card from archive.
-     * 
-     * @param {String} newContent 
-     */
     setSentBackCard(newContent) {
         this.content = newContent;
         for (var i = this.owner.cards.length; i--;) {
             if (this.owner.cards[i].name === this.content) {
                 this.owner.cards[i].isArchived = false;
             }
         }
+    }
+
+    setRecycleBinScreen() {
+        this.archiveScreen = false;
+        this.searchScreen = false;
+        this.recycleBinScreen = true;
```

Comment:
Doesn't this feel awkward? At any given time there can only be one of these which is true, and the others will always be false. Why don't you keep this as a single enum variable like `this.screen = 'recycle bin'`?